etc/services/evio2hipo.yaml

```diff
@@ -6,11 +6,15 @@ io-services:
     class: org.jlab.io.clara.HipoToHipoTagWriter
     name: HipoToHipoTagWriter
 services:
+  # FIXME: replace with dummy/useful service
   - class: org.jlab.service.eb.EBTBEngine
     name: EBTB
 configuration:
-  services:
-    EBTB:
-      dropBanks: 'true'
+  io-services:
+    reader:
+      variation: default
+      timestamp: 05/11/2025-00:00:00
```

Comment:
Should timestamp and schema_dir be commented out?

Reply:
Not sure, but it serves as an example of setting a timestamp during decoding